### PR TITLE
Make Py3 compatible. [3]

### PR DESCRIPTION
**Pull request by amrsoll**
I mainly disabled analytics by default to debug easily.

### CODE REVIEW

**Comment by amrsoll** [comment]
Also I'm currently using this for BeamOS

I know the newer version of the LED driver doesn't have the spread spectrum feature, we might want to talk about this.

**Review by andywerner — CHANGES_REQUESTED** [review]
Please do not disable analytics! How would it get enabled?
If the user did not agree to analytics no analytics data get written nor uploaded by our plugin. But in this case it's required that the analytics data are sent to the plugin in any case.

**Review by andywerner — CHANGES_REQUESTED** [review]
make it work without config file and enable analytics

**Comment by amrsoll** [comment]
TODO : create if branches / try catch to differentiate between py2 and py3, because the neopixel lib and functions have changed.
See 5d4acbd

File: mrbeam_ledstrips/client.py [hunk 1]
```diff
@@ -5,34 +5,33 @@
...
+		print(("Unable to connect to: %s. Daemon running?" % socket_file))
 		sys.exit(1)
 
 	try:
-		s.send(state+'\x00')
-		print "> " + state
-		data = s.recv(5000)
-		print "< " + data
-		s.close()		
+		# s.sendall(b'info')
```
Comment (andywerner) [inline mrbeam_ledstrips/client.py]: please check

**Review by andywerner — APPROVED** [review]
please look at the comment

**Pull request merged**